axum-macros/tests/typed_path/fail/not_deserialize.stderr

```diff
@@ -0,0 +1,8 @@
+error[E0277]: the trait bound `for<'de> MyPath: serde::de::Deserialize<'de>` is not satisfied
+ --> tests/typed_path/fail/not_deserialize.rs:4:14
+  |
+4 | #[typed_path("/users/:id")]
+  |              ^^^^^^^^^^^^ the trait `for<'de> serde::de::Deserialize<'de>` is not implemented for `MyPath`
+  |
+  = note: required because of the requirements on the impl of `serde::de::DeserializeOwned` for `MyPath`
+  = note: required because of the requirements on the impl of `FromRequest<B>` for `axum::extract::Path<MyPath>`
```

Comment:
I think it would make a little more sense for the span here to be the call-site span (point to the derive, not the path inside the `typed_path` attribute).

Reply:
Fixed!